addon/src/-private/array.ts

```diff
@@ -55,6 +55,7 @@ function convertToInt(prop: number | string | symbol): number | null {
   return num % 1 === 0 ? num : null;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unsafe-declaration-merging
```

Comment:
It seems to be upset about this line https://github.com/tracked-tools/tracked-built-ins/pull/421/files#diff-4675cdfe75e78e757eea8a2f32008948d1713d77a78e3dff46f8dec79f297305L214

but I'm not sure it can be removed or why it's there